docs/spec/crisis/README.md

```diff
@@ -0,0 +1,82 @@
+# WIP 
+
+this spec is WIP and simply an overview of the work to be done in the PR.
+Here is the current thinking as to how the Invariant-Broken-Tx PR will be operate:
+
+Within each module invariant registration functions are defined, for example: 
+
+``` golang
+func RegisterInvariants(ak auth.AccountKeeper, ck types.CrisisKeeper) {
+	ck.RegisterInvariantRoute(ModuleName +"/nonnegative-balance",NonnegativeBalanceInvariant(ak))
+	ck.RegisterInvariantRoute(ModuleName +"/some-other-invariant",SomeOtherInvariant(ak))
+}
+```
+
+Within `cmd/gaia/app/app.go` function `NewGaiaApp` the crisis keeper would be
+initialized with all of the invariants from other modules.  
+
+XXX QUESTION:
+because the invariants are registered in `NewGaiaApp` everything would get
```

Comment:
yes, and we probably don't want to store them in state anyways, so we can change them in a non-breaking fashion

Reply:
thanks